docs/application/toc_all.md

```diff
@@ -985,6 +985,7 @@
 #### [Overview](/application/native/guides/location-sensors/overview.md)
 #### [Geofences](/application/native/guides/location-sensors/geofences.md)
 #### [Location Information](/application/native/guides/location-sensors/location.md)
+#### [User Awareness](/application/native/guides/location-sensors/user-awareness.md)
```

Comment:
Question to the location of "User Awareness".

According to the overview.md, **User Awareness** is displayed under **Device Sensors**.
However in this menu tree, 
**User Awareness** is displayed between **Location Information** and **Maps and Maps Service**.
Isn't it better to place the page in the same location with the overview?

Reply:
Hello @anujk-singh , hope you're well. 
Could you please move **line# 988** to **line# 995**, and also include line-gaps above and below the line respectively? 
This is to reposition **line# 988** in the toc_all.md file in order to match it with the sequence as in the overview.md file. 
Please refer to the above comment, and let me know if you have any confusion regarding this. Thank you.